src/synthesize.jl

```diff
@@ -131,8 +138,10 @@ function synthesize(atm::ModelAtmosphere, linelist, λs::AbstractRange; metallic
         α_cntm_layer = LinearInterpolation(cntmλs, α_cntm_vals)
         α[i, :] .= α_cntm_layer.(λs)
 
-        α5[i] = total_continuum_absorption([c_cgs/5e-5], layer.temp, layer.electron_number_density,
-                                           n_dict, partition_funcs)[1]
+        if ! bezier_radiative_transfer
+            α5[i] = total_continuum_absorption([c_cgs/5e-5], layer.temp, 
+                               layer.electron_number_density, n_dict, partition_funcs)[1]
```

Comment:
```suggestion
                                               layer.electron_number_density, n_dict,
                                               partition_funcs)[1]
```
Minor whitespace nitpick